src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -14538,7 +14543,7 @@ void VRFCondenserEquipment::VRFOU_CalcCompH(
     NumOfCompSpdInput = this->CompressorSpeed.size();
     CompEvaporatingPWRSpd.dimension(NumOfCompSpdInput);
     CompEvaporatingCAPSpd.dimension(NumOfCompSpdInput);
-    Q_evap_req = TU_load + Pipe_Q - Ncomp;
+    Q_evap_req = TU_load + Pipe_Q;
```

Comment:
This also looks like it was a bug.

Reply:
I'll double check

Reply:
reverted it back to `Q_evap_req = TU_load + Pipe_Q`